txclib/wizard.py

```diff
@@ -112,7 +112,7 @@ def get_formats(self, filename):
             formats = self.api.get('formats')
         except Exception as e:
             logger.error(e)
```

Comment:
I would prefer logger.error(msg, exc_info=True)` with `msg` describing what the code was trying to do when it got the exception. If you're worried about displaying tracebacks then maybe we should turn off logging when the user is not requesting.